scipy/spatial/tests/test_kdtree.py

```diff
@@ -1106,6 +1106,58 @@ def test_ckdtree_memuse():
     # ideally zero leaks, but errors might accidentally happen
     # outside cKDTree
     assert_(num_leaks < 10)
+
+def test_ckdtree_weights ():
```

Comment:
remove space before ``()`` (this causes the one issue with the TravisCI build right now, not pep8)